### DISCUSSION TITLE
Clone repository

### DISCUSSION THREAD

**Discussion by Tunklj** [recodehive/Stackoverflow-Analysis#465] [Q&A] [answered]
How to clone a repository to my computer?, im new to github

**Comment by quoc10973** [comment, marked as answer]
Go to the GitHub repository page, click the green Code button, and copy the HTTPS or SSH URL.

Clone the Repository:
Open a terminal or command prompt.
Run this command (replace with the copied URL):
- git clone [repository-url]


Access the Cloned Files:
Navigate into the cloned repository folder:
- cd repository-name

You're Ready to Work
